packages/docusaurus-1.x/lib/build-files.js

```diff
@@ -13,10 +13,13 @@ require('@babel/register')({
   only: [__dirname, `${process.cwd()}/core`],
   plugins: [
     require('./server/translate-plugin.js'),
-    '@babel/plugin-proposal-class-properties',
-    '@babel/plugin-proposal-object-rest-spread',
```

Comment:
is the default really needed ?

Reply:
It seems to crash otherwise, something like "default is not a valid property for a plugin".

Reply:
Yes because the modules are coded to be used with es6 imports syntax when babel requires commonJS imports.